.github/mcp/mcp_pytest.py

```diff
@@ -83,28 +83,22 @@
 
     # Create run
     run = create_run(config)
-    print(f'Run created: {run.name}')
+    print(f'[GHA] Run created: {run.name}')
 
     # Wait until run starts before fetching logs
     run = wait_for_run_status(run, status='running')
-    print('Run started. Waiting for run to complete...')
-
-    # Wait up to args.timeout seconds for run to complete
-    try:
-        run = wait_for_run_status(run, status='completed', timeout=args.timeout)
-    except TimeoutError:
-        print(f'Run timed out and did not complete in {args.timeout/60} minutes.')
-
-    # Get run status and stop run
-    success = run.status == RunStatus.COMPLETED
-    print(f'Run completed with status: {run.status} (success={success})')
-    if run.status == RunStatus.RUNNING:
-        stop_runs([run])
-        print('Run stopped.')
```

Comment:
Are we sure we want to relax this try/except?  Can we put it around `follow_run_logs` below and if that fails revert back to polling for status?

Reply:
So the try except is only for the timeout error which I set -- it does not catch the 500 error MAPi was occasionally sending. I think it's fine to not include it